### PR TITLE
Switched CGNS instructions to use CMake

### PR DESCRIPTION
**Pull request by joanibal**
## Purpose
Add more information about how to pass additional flags to ./configure to successfully build cgnstools. 

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

### CODE REVIEW

File: installInstructions/install3rdPartyPackages.rst [hunk 1]
```diff
@@ -325,6 +325,14 @@ Note that the version of these libraries might be different on your machine :
...
 
+If the tk library is install but not found, you may need to add the include library to the CFLAGS used by make. 
+Likewise, if during compilation the linker cannot find definitions for sin and cos, you will also need to include the missing math library by passing ``-lm``.
+The ./configure script in CGNS looks at the environment variables to add user flags, so the final call to ./configure should look like: 
+
+.. prompt:: bash
+   
+   CFLAGS="-I/usr/include/tk -lm" LIBS="-lm" ./configure <....the rest>
+
 .. warning::
```
Comment (nwu63) [inline installInstructions/install3rdPartyPackages.rst:336]: Unrelated but could you remove this block since it's for 3.3.0 which is no longer supported?

Comment (sseraj) [reply installInstructions/install3rdPartyPackages.rst:336]: Aren't we still using 3.3.0 for stable?

Comment (nwu63) [reply installInstructions/install3rdPartyPackages.rst:336]: Oh shoot we haven't removed 3.3.0 yet, okay let's do that when we roll `stable` to 4.1.2.

**Review by nwu63 — APPROVED** [review]

**Comment by eirikurj** [comment]
While this change is fine, it seems that autotools is more cumbersome compared to cmake as it is able to find tk properly. I am starting to lean towards cmake again as there are other issues that show up that need to be addressed similarly, making the overall install process more complicated than it should be.

**Comment by marcomangano** [comment]
This looks good! I cannot test it on my new machine until I am back, but this should be the definitive fix, right @nwu63 ?

**Review by nwu63 — CHANGES_REQUESTED** [review]
Actually, we recently changed the docker build back to using `cmake` since it fixed a number of issues (such as the workaround in this PR). We should probably update this to reflect that.

**Review by nwu63 — APPROVED** [review]

**Pull request merged**